### PR TITLE
Update rusefi_config.txt  - correct first obsolete RE wiki link from #235

### PR DESCRIPTION
**Pull request by mi-hol**

### CODE REVIEW

**Review by nmschulte — APPROVED** [review]

File: firmware/integration/rusefi_config.txt [hunk 1]
```diff
@@ -1098,7 +1098,7 @@ int16_t tps2Max;;"ADC", 1, 0, 0, 1023, 0
 	custom InjectionTimingMode 1 bits, U08, @OFFSET@, [0:1], "End of injection", "Start of injection", "Center of injection"
 	InjectionTimingMode injectionTimingMode;Sets what part of injection's is controlled by the injection phase table.
 
-	debug_mode_e debugMode;See http://rusefi.com/s/debugmode
+	debug_mode_e debugMode;See https://wiki.fome.tech/r/debugmode
```
Comment (nmschulte) [inline firmware/integration/rusefi_config.txt:1101]: Q: What provides this `r/debugmode` feature of the Wiki site (it redirected me to https://wiki.fome.tech/Advanced-Features/Debug-Mode/ -- cool)?

Comment (nmschulte) [reply firmware/integration/rusefi_config.txt:1101]: Oh I see, thanks!

https://github.com/FOME-Tech/fome-fw/issues/235#issuecomment-1715299377

Comment (karniv00l) [reply firmware/integration/rusefi_config.txt:1101]: I've also left some info in the README about this feature: https://github.com/FOME-Tech/wiki?tab=readme-ov-file#redirection

Comment (mi-hol) [reply firmware/integration/rusefi_config.txt:1101]: > info in the README about this feature: https://github.com/FOME-Tech/wiki?tab=readme-ov-file#redirection

this was exactly the place where I found this info :)

**Pull request merged**